cmake/cpu_features.cmake

```diff
@@ -36,7 +36,8 @@ if (ARCH_AARCH64)
     # [1] https://en.wikipedia.org/wiki/AArch64
     option (TIFLASH_ENABLE_ASIMD_SUPPORT "Enable Advanced SIMD support." ON)
     option (TIFLASH_ENABLE_SVE_SUPPORT "Enable Scalable Vector Extension support." OFF)
```

Comment:
`TIFLASH_ENABLE_ASIMD_SUPPORT` is used as default feature in v6.3.

Reply:
fixed